orbit/diagnostics/plot.py

```diff
@@ -293,14 +293,12 @@ def plot_posterior_params(mod, kind='density', n_bins=20, ci_level=.95,
 
     posterior_samples = deepcopy(mod._posterior_samples)
 
-    if len(mod._positive_regressor_col) > 0:
-        for i, regressor in enumerate(mod._positive_regressor_col):
-            posterior_samples[regressor] = posterior_samples['pr_beta'][:, i]
-    if len(mod._regular_regressor_col) > 0:
-        for i, regressor in enumerate(mod._regular_regressor_col):
-            posterior_samples[regressor] = posterior_samples['rr_beta'][:, i]
-
-    params_ = mod._positive_regressor_col + mod._regular_regressor_col + ['obs_sigma']
+    if len(mod._regressor_col) > 0:
+        for i, regressor in enumerate(mod._regressor_col):
+            posterior_samples[regressor] = posterior_samples['beta'][:,i]
+
+    params_ = mod.regressor_col + ['obs_sigma']
```

Comment:
when there is no regression, `mod.regressor_col` is None, this would trigger an error here. either we use `mod._regressor_col` here  or add `if mod.regressor_col is not None`

Reply:
good catch! Fixed it